applications/launchpad/gui-react/src/components/TabContent/index.tsx

```diff
@@ -19,7 +19,7 @@ const TabContent = ({
     <StyledTabContent>
       <TabMainText>{text}</TabMainText>
       {running && !pending ? (
-        <Tag variant='small' type='running' subText={tagSubText}>
+        <Tag variant='small' type='running' subText={tagSubText} dashboard>
```

Comment:
NIT: Not sure if `dashboard` is a good name here... I would rather try to keep things in `components/*` generic as much as possible. So instead of using a _reason_ as a prop name, you can name the prop after the _effect_ that it applies. For example, if something on the dashboard should be large, then the prop should be named `large` rather than `dashboard`.

Reply:
Naming improved.